### DISCUSSION TITLE
Publish a Bill of Material for Spark?

### DISCUSSION THREAD

**Discussion by SimonMarquis** [leboncoin/spark-android#192] [Decision making]
To be able to bundle modules with the same versions.

```kotlin
dependencies {
  implementation(platform("com.adevinta.spark:spark-bom:x.y.z")
  implementation("com.adevinta.spark:spark")
  implementation("com.adevinta.spark:spark-icon")
}
```

https://docs.gradle.org/current/userguide/java_platform_plugin.html#sec:java_platform_publishing

**Comment by soulcramer** [comment]
Yes that would be nice since we'll  have more libs in the future (illustrations for ex)

**Comment by soulcramer** [comment]
What format should follow? The compose one with the release date?

**Reply by SimonMarquis** [reply]
No, I think we should keep things simple. Using the same version name as in the snippet from the description.

**Comment by SimonMarquis** [comment]
- #282
